frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -65,13 +65,13 @@
             </v-col>
             <v-col class="v-col-12 v-col-md-9 v-col-xl-11">
               <v-select
-                v-if="isAnAccountMaintenanceRequest"
+                v-if="isAnAccountMaintenanceRequest || isAnIrregularExpenseRequest"
                 id="selectFacility"
                 v-model="newRequestModel.facilities"
                 placeholder="[select facility]"
                 variant="outlined"
                 density="compact"
-                :items="facilities"
+                :items="isAnIrregularExpenseRequest ? filterFacilitiesWithoutFA : facilities"
```

Comment:
If you made this a computed method (filteredFacilities) it would make this attribute simpler and you wouldn't need the additional filterFacilitiesWithoutFA method as the logic could just be in there. I.e. if not irregular return facilties else filter by fa.